integration-tests/LDDFeatureRequesterTest.php

```diff
@@ -40,26 +40,61 @@ public function testGetApc() {
 
         # cached value so not updated
         $redis->hset("launchdarkly:features", 'foo', $this->gen_feature("foo", "baz"));
-        $this->assertEquals("bar", $client->variation('foo', $user, 'jim'));
+        $this->assertEquals("baz", $client->variation('foo', $user, 'jim'));
```

Comment:
ok, i now know that this change is not acceptable. i'm having trouble understanding why it seems to be necessary in my vagrant test-suite. closing this merge request. i'll focus on getting the integration tests to run in CI instead.